.babelrc

```diff
@@ -29,19 +30,22 @@
         ]
       ]
     },
-    "test": {
+    "rollup": {
```

Comment:
We are mixing concerns here a little. Target and tool we're using for bundling which feels odd.

Reply:
I would like 3 environments as well. But doing that opens an entire can of worms. I also feel this naming could be better. Do you have any suggestions maybe?